scripts/test/index.js

```diff
@@ -0,0 +1,9 @@
+const { spawn } = require('child_process')
+const { inputDir = './source' } = require('yargs').argv
+
+const args = [
+  require.resolve('mocha/bin/_mocha'),
+  `${inputDir}/**/__tests__/*-test.js`
```

Comment:
So you shouldn't have to do this anymore:
https://github.com/everydayhero/warp-gate/blob/master/ui/.babelrc